spec/middleware/rack/attack_spec.rb

```diff
@@ -119,6 +119,37 @@ def app
     end
   end
 
+  describe 'facilities_va/ip' do
+    let(:endpoint) { '/facilities_api/v1/va' }
+    let(:headers) { { 'X-Real-Ip' => '1.2.3.4' } }
+    let(:limit) { 30 }
+
+    before do
+      limit.times do
+        get endpoint, nil, headers
+        expect(last_response.status).not_to eq(429)
+      end
+
+      get endpoint, nil, other_headers
+    end
+
+    context 'response status for repeated requests from the same IP' do
+      let(:other_headers) { headers }
+
+      it 'limits requests' do
+        expect(last_response.status).to eq(429)
+      end
+    end
+
+    context 'response status for request from different IP' do
+      let(:other_headers) { { 'X-Real-Ip' => '4.3.2.1' } }
+
+      it 'limits requests' do
```

Comment:
**Nitpick:** it looks like this test is confirming the requests don't get limited, so this should probably just be changed to `'does not limit requests'`

Reply:
Yep, that's true, will change that message

Reply:
done